Vagrantfile

```diff
@@ -92,6 +92,7 @@ Vagrant.configure(2) do |config|
         vb.gui = false
         vb.customize ['modifyvm', :id, '--memory', memory]
         vb.customize ['modifyvm', :id, '--cpus', cpu]
+	vb.customize ['modifyvm', :id, '--uartmode1', 'disconnected']
```

Comment:
It seems the indentation is not correct

Reply:
![image](https://user-images.githubusercontent.com/15902071/57572822-85e95800-743d-11e9-8dcc-77f650a290e7.png)

Looks okay to me. Did you get an error while running the Vagrantfile?

Reply:
I think tabs and spaces are mixed. 
![2019-05-11_23-22](https://user-images.githubusercontent.com/1148405/57573327-a3b9bb80-7443-11e9-9bc0-2fe8a9f86d13.png)